client/my-sites/importer/importing-pane.jsx

```diff
@@ -237,6 +237,7 @@ class ImportingPane extends React.PureComponent {
 						sourceAuthors={ customData.sourceAuthors }
 						sourceTitle={ customData.siteTitle || translate( 'Original Site' ) }
 						targetTitle={ siteName }
+						sourceType={ this.props.title }
```

Comment:
I can see the prop name `title` confusing somebody somewhere down the line, but guess it might not be easy to change.

Reply:
I agree, I've made it so it gets passed in to here as `sourceType` now. Further up, it gets a bit trickier as it is more of a title.